lemmy_api/src/lib.rs

```diff
@@ -100,6 +100,23 @@ pub(in crate) async fn check_community_ban(
   }
 }
 
+pub(in crate) async fn collect_moderated_communities(
```

Comment:
Would be nice if you could add some quick rustdoc here (and for other functions). And yes I know we are missing docs almost everywhere...

Reply:
i added some in a few areas, i'm out of steam tonght 😴